webapp/handlers.py

```diff
@@ -103,13 +122,99 @@ def internal_error(error):
             app.extensions["sentry"].captureException()
 
         return (
-            flask.render_template("50X.html", error_name=error_name),
+            render_template("50X.html", error_name=error_name),
             return_code,
         )
 
-    @app.errorhandler(503)
-    def service_unavailable(error):
-        return flask.render_template("503.html"), 503
```

Comment:
There is no need to delete this one, this is to show a nice page whenever a 503 happens in the entire application.

Did you remove it for any reason in particular?

Reply:
I removed it because I though the same 503 exception has been handled with this. 
`    @app.errorhandler(StoreApiConnectionError)
      def connection_failed(error):
        return render_template("503.html"), 503`

Reply:
I see. It's a different exception, `StoreApiConnectionError` will be raised only for the store API module, whoever the `503` exception can happen by many different things. But we indeed want to show the same template for both exceptions, so one solution could be:

```python
    @app.errorhandler(503)
    @app.errorhandler(StoreApiConnectionError)
    def connection_failed(error):
        return render_template("503.html"), 503
```

Reply:
Ok, Great! Thank you Fran. I will resolve that